nservicebus/gateway/scale-out.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Gateway And Scale-Out
+summary: Explanation of how the Gateway handles Scale-Out
+tags: []
+related:
+ - samples/gateway
+---
+
+The gateway is designed to run on a single instance for each logical endpoint. A scaled out endpoint needs one instance to receive all the incoming HTTP gateway traffic for that endpoint. There are different strategies to achieve this based on the transport and the scale-out mechanism utilized:
+
+### MSMQ with the distributor
+
+With MSMQ scale out using the [distributor](/nservicebus/scalability-and-ha/distributor), the gateway runs on the master node/distributor, making it the natural single receiving gateway instance for the endpoint. Running the master node/distributor as [a clustered service on a Windows Failover Cluster](/nservicebus/scalability-and-ha/deploying-to-a-windows-failover-cluster.md) provides High Availability(HA). No special action must be taken other than configuring senders to send gateway HTTP traffic to the master node instead of directly to the workers.
+
+### Others - Brokered transports and MSMQ scaled out with new unified scalability model in version 6
+
+With [brokered transports](/nservicebus/scalability-and-ha/scale-out.md#versions-5-and-below-sql-server-and-rabbitmq) or the [new unified scalability model](/nservicebus/scalability-and-ha/sscale-out.md#versions-6-and-above) in version 6, there is no natural single receiving gateway instance for each logical endpoint. One needs to be explicitly selected and any endpoints sending to a gateway must be configured to use the HTTP address of the selected instance.
+
+Place a HTTP Load Balancer(LB) in front to avoid hard coding individual endpoint instance gateway HTTP addresses in gateway senders. Configure the LB to forward all traffic to the HTTP address of the selected endpoint instance and sending endpoints to send through the LB. 
+
```

Comment:
NOTE: The gateway might be running in each instance but only a single one can be configured to be used by the gateway on the other side. If the scaled out endpoint communicates with more than one site, each site can point to a different gateway instance within the cluster.